newm-server/src/main/resources/application.conf

```diff
@@ -80,3 +80,15 @@ sentry {
 cloudinary {
     url = ${CLOUDINARY_URL}
 }
+
+aws {
+    accessKeyId = ${AWS_ACCESS_KEY_ID}
+    secretKey = ${AWS_SECRET_KEY}
+    region = ${AWS_REGION}
+    s3 {
+        audioUpload {
+            bucketName = ${AWS_AUDIO_BUCKET}
+            timeToLive = 180
```

Comment:
Is the time in days? Why do we limit this?

Reply:
Seconds. Needs to be short for security reasons.